src/mote/templates/mainpage.html

```diff
@@ -394,5 +395,34 @@ <h6 class="modal-title head fw-bold">Recent conversations</h6>
                 </div>
             </div>
         </div>
+        <div class="toast-container"> 
+            <div class="position-fixed bottom-0 end-0 p-3" style="z-index: 11">
+                <div id="liveToast" class="toast" role="alert" aria-live="assertive" aria-atomic="true">
+                    <div class="toast-body">
+                        A meeting has just ended
```

Comment:
I think we need a little more information in this notification. The meeting name at least.